content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -226,15 +259,16 @@ secrets:
 - name: default-token-uudge
 ```
 
-Using your editor of choice (for example `vi`), open the `sa.yaml` file, delete line with key `resourceVersion`, add lines with `imagePullSecrets:` and save.
+Using your editor, delete the line with key `resourceVersion`, add lines for `imagePullSecrets:` and save it.
+Leave the `uid` value set the same as you found it.
 
-The output of the `sa.yaml` file is similar to this:
+After you've made those changes, the content of your `sa.yaml` will look something like this:
```

Comment:
nit: This is the first time `sa.yaml` is mentioned. When is the output saved to sa.yaml?